ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -10,7 +10,8 @@
 from ocs_ci.ocs.resources.packagemanifest import get_selector_for_ocs_operator, PackageManifest
```

Comment:
its not really part of your PR, but plz fix the import ordering in this file

Reply:
Fixed